eva-server/src/main/java/uk/ac/ebi/eva/server/security/authorization/Oauth2Configuration.java

```diff
@@ -0,0 +1,24 @@
+package uk.ac.ebi.eva.server.security.authorization;
+
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.oauth2.config.annotation.web.configuration.EnableResourceServer;
+import org.springframework.security.oauth2.config.annotation.web.configuration.ResourceServerConfigurerAdapter;
+
+
+@ConditionalOnProperty(prefix = "eva.server.security", name = "enabled", havingValue = "true")
+@Configuration
+@EnableResourceServer
+public class Oauth2Configuration extends ResourceServerConfigurerAdapter {
```

Comment:
Could a bit of documentation for enabling the OAuth server (even just a link to Spring docs) be added to the README file?